payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -260,9 +260,22 @@ internal class ElementsSessionJsonParser(
         }
     }
 
-    private fun parseCardBrandChoiceEligibility(json: JSONObject): Boolean {
-        val cardBrandChoice = json.optJSONObject(FIELD_CARD_BRAND_CHOICE) ?: return false
-        return cardBrandChoice.optBoolean(FIELD_ELIGIBLE, false)
+    private fun parseCardBrandChoice(json: JSONObject): ElementsSession.CardBrandChoice? {
+        val cardBrandChoice = json.optJSONObject(FIELD_CARD_BRAND_CHOICE) ?: return null
+        val preferredNetworks = mutableListOf<String>()
+
+        cardBrandChoice.optJSONArray(FIELD_PREFERRED_NETWORKS)?.let { jsonArray ->
```

Comment:
Can you add a test that exercises this code path? One with the card brand choice field, but no preferred networks.